### PR TITLE
feat: add Ukrainian translation

### PR DESCRIPTION
**Pull request by Zlatex**
## Description

Added new translation: Ukrainian.

Could you please leave screenshots where these values are used so I know if I translated everything correctly:

1. logoutUser
2. loginWithAnotherUser
3. loginUser
4. resetPasswordExpiration
5. openInNewTab
6. relatedDocument
7. toggleBlock
8. filterWhere
9. validUploadID
10. currentDocumentStatus

I also have a question to "missingRequiredData". Is it the same as missing mandatory data? 

- [x] I have read and understand the CONTRIBUTING.md document in this repository

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Zlatex** [comment]
Got the following errors when running tests, I doubt they are related to my changes.
image

File: src/translations/ua.json [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    "stayLoggedIn": "Залишатись в системі",
+    "successfullyUnlocked": "Успішно розблоковано",
+    "unableToVerify": "Неможливо підтвердити",
+    "verified": "Підтверджено",
+    "verifiedSuccessfully": "Успішно Підтверджено",
+    "verify": "Підтвердити",
+    "verifyUser": "Підтвердити користувача",
+    "verifyYourEmail": "Підтвердити пошту",
+    "youAreInactive": "Ви були неактивні певний час і скоро, в цілях вашої безпеки, вас буде розлогінено. Чи бажаєте ви залишитись в системі?",
+    "youAreReceivingResetPassword": "Ви отримали це повідомлення, бо ви (або хтось інший) створив запит на скидання пароля до вашого облікового запису. Будь ласка, натисніть на наступне посилання, або вставте посилання в адресну строку вашого браузера, щоб завершити процес:",
```
Comment (gesha17133) [inline src/translations/ua.json:56]: "рядок адреси" would be better, wouldn't it?

File: src/translations/ua.json [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    "leaveWithoutSaving": "Вийти без збереження",
+    "light": "Світла",
+    "loading": "Загрузка...",
+    "locales": "Переклади",
+    "moveDown": "Перемістити нижче",
+    "moveUp": "Перемістити вище",
+    "newPassword": "Новий пароль",
+    "noFiltersSet": "Відсусті фільтри",
+    "noLabel": "<без {{label}}>",
+    "noResults": "Жодного {{label}} не знайдено. Або {{label}} ще не існує, або жодна з них не відповідає фільтрам які ви задали више.",
```
Comment (gesha17133) [inline src/translations/ua.json:187]: ...фільтрам, які ...

**Review by Xanndr-exe — COMMENTED** [review]
Great translation!!! Thanks for the Ukrainian Language, Which may be helpful for the local market and further usage, left a few gramma comments

File: src/translations/ua.json (hunk 1 above)
Comment (Zlatex) [reply src/translations/ua.json:56]: I think we can make "або вставте посилання в адресний рядок вашого браузера" or just  "або вставте посилання в ваш браузер", what do you think?

File: src/translations/ua.json (hunk 2 above)
Comment (Zlatex) [reply src/translations/ua.json:187]: Thanks, I'll change

**Comment by DanRibbens** [comment]
I reviewed it using Google translate and it looks good.

It is a bit difficult to grab all the screenshots you requested. If there are problems with any translations, you or somebody else can make another PR.

Thank you so much for this!

**Review by DanRibbens — APPROVED** [review]

**Pull request merged**